end-to-end-tests/specs/core/mutationTable.spec.js

```diff
@@ -51,11 +51,12 @@ describe('Mutation Table', function() {
         it('should show more exon number after clicking "Show more"', ()=>{
             // click "show more" to add more data
             browser.click("button*=Show more");
+
+            //browser.debug();
             // check if 6 exact matches for 6 appear
+
             browser.waitUntil(() => {
-                textValuesWith6 = browser.getText('//*[text()="6"]');
-                // 4 values from Exon column + 1 value from #Mut in Sample column + 1 value from COSMIC column
-                return textValuesWith6.length === 6;
+                return testJqueryExpression( ()=>$("[class*=exon-module__exon-table]").length) == 38;
```

Comment:
@adamabeshouse this is helper which i think you would like.   allows you to use full power of jquery selectors instead of crappy webdriverio selectors

Reply:
oooh cool. how about

```
 testJqueryExpression(expression:string) {
           return browser.execute(()=>$("[class*=exon-module__exon-table]").length).value;
}
```